install/infra/modules/eks/kubernetes.tf

```diff
@@ -95,18 +129,18 @@ module "eks" {
 
     Workspaces = {
       instance_types = [var.workspace_machine_type]
-      name = "ws-${var.cluster_name}"
-      subnet_ids   = module.vpc.public_subnets
-      min_size     = 1
-      max_size     = 10
-      block_device_mappings =  [{
+      name           = "ws-${var.cluster_name}"
+      subnet_ids     = module.vpc.public_subnets
+      min_size       = 1
+      max_size       = 10
```

Comment:
We are planning to change the max node pool size here to 50, see https://github.com/gitpod-io/website/pull/2554